lambda/getProductsList/index.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  try {
+    // Mock data for the products array
+    const products = [
```

Comment:
You are using the same data for 2 files. Create a separate file and import data from there